src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -39,6 +44,10 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         this.userPrefs = new UserPrefs(userPrefs);
         sortedPersons = new SortedList<>(this.addressBook.getPersonList());
         filteredPersons = new FilteredList<>(sortedPersons);
+        // to add comparator for sorting logic here
+        observableAppointments = FXCollections.observableArrayList();
+        sortedAppointments = new SortedList<>(observableAppointments,
```

Comment:
Maybe instead of holding Appointment objects, we can use another class which provides additional information like the contact name for the sidebar

Reply:
Currently considering creating bidirectional associations between Person and Appointment classes.